contribs/analysis/src/main/java/org/matsim/contrib/analysis/vsp/traveltimedistance/TravelTimeValidationRunnerPreAnalysis.java

```diff
@@ -29,6 +29,10 @@
     private final TravelTimeDistanceValidator validator;
     private final String outputFolder;
 
+    private double insideTripProportion = 0.6;  // Proportion of trips will be chosen within the key area (when key area shape file is provided)
```

Comment:
Can be static final and with javadoc tag instread `/** ... **/`

Reply:
I saw you made them configurable, so never mind the static final. But still proper javadoc tag is more preferable.